app/questionnaire/questionnaire_schema.py

```diff
@@ -94,11 +95,37 @@ def __init__(
         self._populate_answer_dependencies()
         self._populate_when_rules_section_dependencies()
         self._populate_calculated_summary_section_dependencies()
+        self._populate_min_max_map()
 
     @cached_property
     def answer_dependencies(self) -> ImmutableDict[str, set[AnswerDependent]]:
         return ImmutableDict(self._answer_dependencies_map)
 
+    def _append_to_min_max_map(
+        self, min_max: Literal["minimum", "maximum"], answer_id: str, answers: list
+    ) -> None:
+        if value := answers[0].get(min_max, {}).get("value"):
+            if isinstance(value, int):
+                self.min_and_max_map[answer_id] = str(value)
+            elif isinstance(value, dict) and value:
+                if (
+                    value.get("source") == "answers"
+                    and value["identifier"] in self.min_and_max_map
```

Comment:
1. When would `value["identifier"]` not be in `self.min_and_max_map`?
2. Why would `value["identifier"]` not be in `self.min_and_max_map`?
This is going to skip it, instead of working it out?

Reply:
You're right, this was added yesterday to get all the schemas to pass. I took it out and investigated it and it turns out there is an MBS schema that has an answer thats referenced that doesn't actually have a maximum value so I think its an issue with that schema that needs to be fixed.